pkgs/development/libraries/dbus/default.nix

```diff
@@ -32,15 +35,22 @@ stdenv.mkDerivation rec {
     # Also applied upstream in https://gitlab.freedesktop.org/dbus/dbus/-/merge_requests/189,
     # expected in version 1.14
     ./docs-reproducible-ids.patch
-  ] ++ (lib.optional stdenv.isSunOS ./implement-getgrouplist.patch);
+  ] ++ (lib.optional stdenv.isSunOS ./implement-getgrouplist.patch)
+    # AC_PATH_XTRA doesn't seem to find X11 libs even though libX11 seems
+    # to provide valid pkg-config files. This replace AC_PATH_XTRA with
+    # PKG_CHECK_MODULES
+    ++ lib.optional stdenv.hostPlatform.isStatic
+      (fetchpatch {
```

Comment:
As noted in the MR, this breaks the `--without-x` flag.

Reply:
Fixed.

Reply:
Fixed cf https://gitlab.freedesktop.org/dbus/dbus/-/merge_requests/212#note_1170616.